<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinProtectionForm</class>
 <widget class="QWidget" name="PinProtectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="pinProtectionHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="pinProtectionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pin Protection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="numberPad" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>130</width>
     <height>175</height>
    </rect>
   </property>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>135</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/backspace.svg</normaloff>:/settings/icons/backspace.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="digit7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit0">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>135</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit8">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit5">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>45</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>45</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">4</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="pinFunctions" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>160</width>
     <height>175</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="pin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff; color: #000000</string>
    </property>
    <property name="placeholderText">
     <string>Enter Pin</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMessage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #cc0000</string>
    </property>
    <property name="text">
     <string>Entered Pin is invalid!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>135</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
